tests/fixtures/minimal-localizable-web-ext/_locales/en/messages.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extensionName": {
+    "message": "Name of the extension",
+    "description": "This is an example of a minimal extension that does nothing"
+  },
+
+  "version": "1.0",
```

Comment:
why does this include `version`, `applications`, and `background`? They are not needed. It only needs `extensionName` or possibly some other localization keys.